mobile-widgets/qmlmanager.cpp

```diff
@@ -98,6 +98,10 @@ void QMLManager::btHostModeChange(QBluetoothLocalDevice::HostMode state)
 	}
 	emit btEnabledChanged();
 }
+void QMLManager::btRescan()
+{
+	BTDiscovery::instance()->BTDiscoveryReDiscover();
+}
```

Comment:
this only gets compiled with BT_SUPPORT enabled
so we should make sure that it only gets called / referenced when BT_SUPPORT is enabled
OR
we should give up on the idea that one could / would want to build Subsurface-mobile without BT_SUPPORT 

(I think the latter is more reasonable)

Reply:
Even if the latter is reasonable, will wrap this with #ifdef's .

Reply:
no, don't.
I'm minutes away from checking in changes that disable no-bt builds for Subsurface-mobile
it's just silly to make our code more complex to support that